packages/design-system/src/components/Icon/constants.js

```diff
@@ -0,0 +1,57 @@
+export const IconSizeName = {
+  XSmall: 'XSmall',
+  Small: 'Small',
+  Medium: 'Medium',
+  Large: 'Large',
+  XLarge: 'XLarge'
+};
+
+export const IconSize = {
+  [IconSizeName.XSmall]: {
+    width: 16,
+    height: 16
+  },
+  [IconSizeName.Small]: {
+    width: 20,
+    height: 20
+  },
+  [IconSizeName.Medium]: {
+    width: 24,
+    height: 24
+  },
+  [IconSizeName.Large]: {
+    width: 32,
+    height: 32
+  },
+  [IconSizeName.XLarge]: {
+    width: 48,
+    height: 48
+  }
+};
+
+export const IconTypeName = {
+  Primary: 'primary',
+  Inverted: 'inverted',
+  Link: 'link',
+  Success: 'success',
+  Warning: 'warning',
+  Error: 'error'
+};
+
+export const IconColorMapper = {
+  [IconTypeName.Primary]: '#424d57',
+  [IconTypeName.Inverted]: '#fff',
+  [IconTypeName.Link]: '#4384f5',
+  [IconTypeName.Success]: '#4bb678',
+  [IconTypeName.Warning]: '#efa842',
+  [IconTypeName.Error]: '#d64646'
+};
```

Comment:
I was also thinking about passing disabled prop (to avoid duplication), but I think if we were to create some other props then different dictionaries seems like a better idea. WDYT?

```suggestion
export const IconColorMapper = disabled => ({
  [IconTypeName.Primary]: disabled ? '#686d72' : '#424d57',
  [IconTypeName.Inverted]: '#fff',
  [IconTypeName.Link]: '#4384f5',
  [IconTypeName.Success]: '#4bb678',
  [IconTypeName.Warning]: '#efa842',
  [IconTypeName.Error]: '#d64646'
});
```